app/javascript/Search/__stories__/SearchForm.stories.jsx

```diff
@@ -21,7 +21,7 @@ class FocusedForm extends Component {
     // Disabling prop types checks here because this is simply a wrapper
     // class for a Storybook story.
     // eslint-disable-next-line react/destructuring-assignment, react/prop-types
-    return this.props.children[0];
+    return toChildArray(this.props.children)[0];
```

Comment:
Preact no longer treats the children prop as an array. If you want it as an array, you need to explicitly convert it. This is how React handles it.